hr_attendance_sheet_compensatory/models/hr_attendance.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 Pierre Verkest
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import SUPERUSER_ID, _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class HrAttendance(models.Model):
+    _inherit = "hr.attendance"
+
+    is_overtime_due = fields.Boolean(
+        string="Is overtime due",
+        default=False,
+        help="Whether the overtime is due or not. "
+        "By default overtime is not due until a manager validates it.",
+    )
+
+    @api.model
+    def create(self, *args, **kwargs):
+        attendance = super().create(*args, **kwargs)
```

Comment:
this feels weird

why isn't it this instead?

```suggestion
    @api.model_create_multi
    def create(self, vals_list):
        attendances = super().create(vals_list)
```